cmake/ctest/drivers/geminga/cron_driver.sh

```diff
@@ -102,32 +121,15 @@ echo "OMPI_CXXFLAGS after $OMPI_CXXFLAGS"
 echo "Configuration = $CTEST_CONFIGURATION"
 env
 
-# export CUDA_LAUNCH_BLOCKING=1
-# export CUDA_MANAGED_FORCE_DEVICE_ALLOC=1
-# Only run on the Tesla K40, not the Quadro
-# export CUDA_VISIBLE_DEVICES=0
-# Machine independent cron_driver:
-# SCRIPT_DIR=`cd "\`dirname \"$0\"\`";pwd`
-$SCRIPT_DIR/../cron_driver.py
-
-#module unload nvcc-wrapper
-#module unload cuda
-#module unload gcc
-#module unload openmpi
-# module unload sems-netcdf/4.7.3/parallel
-# module unload sems-superlu/4.3
-# module unload sems-cuda_openmpi/4.0.2/base
-# module unload sems-cuda/10.1
-# module unload sems-openmpi/4.0.2
-# module unload sems-zlib/1.2.8/base
-# module unload sems-python/2.7.9
-# module unload sems-boost/1.69.0/base
-# module unload sems-gcc/8.3.0
-# module unload sems-cmake/3.17.1
-# module unload sems-env
+pushd $TRILINOS_SOURCE
+ctest -S $BUILDS_DIR/ctest_linux_nightly_mpi_release_muelu_kokkos_refactor_cuda_geminga.cmake
+ctest -S $BUILDS_DIR/ctest_linux_nightly_mpi_release_muelu_amgx_cuda_geminga.cmake
+ctest -S $BUILDS_DIR/ctest_linux_nightly_mpi_release_muelu_cuda_no_uvm_geminga.cmake
```

Comment:
Is this to avoid python2 dependencies?

Reply:
Yep. According to @bartlettroscoe, the dashboard driver isn't really maintained.